source/services/odnoklassniki.js

```diff
@@ -7,9 +7,7 @@ var config = require('../config'),
     dom    = require('../dom');
 
 var odnoklassniki = {
-    counterUrl: config.secure 
-        ? undefined 
-        : 'http://connect.ok.ru/dk?st.cmd=extLike&ref={url}&uid={index}',
+    counterUrl: config.protocol + '://connect.ok.ru/dk?st.cmd=extLike&ref={url}&uid={index}',
```

Comment:
@iamakulov Okay, I get it. Seems we should be consistent here and I'll make an issue about getting rid of `config.protocol` and make a pr for it.